mta-sts.md

```diff
@@ -391,50 +388,35 @@ not "example.com".
 When sending to an MX at a domain for which the sender has a valid and
 non-expired MTA-STS policy, a sending MTA honoring MTA-STS MUST validate:
 
-1.  That the recipient MX supports STARTTLS and offers a valid PKIX-based TLS
-    certificate.
+1. That at least one of the policy's "mx" patterns matches the selected MX host,
+   as described in (#mx-host-validation), "MX Host Validation".
```

Comment:
Should "testing" be mentioned here?  When testing these MUST conditions pertain only to problem reporting, connections are made regardless and delivery proceeds as though MTA-STS policy were not place...

Reply:
I hope not to, because the section "Policy Application" goes into more detail on what to do when a policy is "invalid."

Note the following sentence:

This section does not dictate the behavior of sending MTAs when policies fail to
validate; see (#policy-application), "Policy Application" for a description of
sending MTA behavior when policy validation fails.

What do you think? Clear enough?

Reply:
Perhaps if the phrase `MUST validate` is changed to `MUST check whether` with small corresponding tweaks to the text below, then it is more clear that these must be checked, but may not need to be enforced.  If you feel strongly it is clear enough, I guess it'll have to do...

Reply:
Done.